css/styles.css

```diff
@@ -0,0 +1,222 @@
+/* $color1: rgba(7, 59, 58, 1);
+$color2: rgba(95, 15, 64, 1);
+$color3: rgba(154, 3, 30, 1);
+$color4: rgba(251, 139, 36, 1);
+$color5: rgba(227, 100, 20, 1); */
+
+/* .try{
+background: rgba(237, 62, 9, 1);
+background: rgba(129, 193, 75, 1);
+color: rgba(46, 147, 60, 1);
+color: rgba(41, 112, 69, 1);
+color: rgba(73, 79, 75, 1);
+} */
```

Comment:
Don't leave unused code as a comment in your codebase. Remove them if they are not needed.